File: frontend/src/services/riskIntelligence.ts
import { apiFetch } from "./api";

export interface RiskIntelligenceData {
  overallRisk: string;
  riskScore: number;
  confidence: number;
  topFinding: string;
  recommendations: string[];
}

export async function getRiskIntelligence(): Promise<RiskIntelligenceData> {
  return apiFetch<RiskIntelligenceData>("/risk-intelligence/");
}

File: backend/app/routers/risk_intelligence.py
from fastapi import APIRouter

router = APIRouter()

@router.get("/risk-intelligence/")
def get_risk_intelligence():
    return {
        "overallRisk": "High",
        "riskScore": 92,
        "confidence": 97,
        "topFinding": "Multiple privileged accounts detected",
        "recommendations": [
            "Enable MFA",
            "Review privileged roles",
            "Disable inactive identities"
        ]
    }